js/src/dom/selectorEngine.js

```diff
@@ -29,17 +31,13 @@ const SelectorEngine = (() => {
   if (!Element.prototype.closest) {
     fnClosest = (element, selector) => {
       let ancestor = element
-      if (!document.documentElement.contains(element)) {
```

Comment:
this polyfill come from MDN, why did you change that ?

Reply:
Tests are failing with that `if`. In any case, the DOM Standard specification (https://dom.spec.whatwg.org/#dom-element-closest) doesn't require the element to be a discendant of the `document` object

Reply:
that's right but I don't see a use case where the element should be outside of the `document.documentElement` :thinking:

Reply:
Probably some tests don't add elements to the DOM before execution. If you prefer, I can revert this change and fix those tests.

Reply:
No that's ok :+1: